R/TealAppDriver.R

```diff
@@ -76,7 +76,7 @@ TealAppDriver <- R6::R6Class( # nolint: object_name.
         strict = FALSE
       )
 
-      required_version <- 105
+      required_version <- "105"
```

Comment:
Can we add a comment explaining the reason for this change? This will help prevent confusion for anyone reviewing it in the future.

Reply:
I don't think so. The `?numeric_version` doc is clear that it only works with character vector.

```r
Arguments:

       x: for the creators, a character vector with suitable numeric
          version strings (see ‘Details’); for ‘package_version’,
          alternatively an R version object as obtained by ‘R.version’.
          For ‘as.numeric_version’ and ‘as.package_version’, suitable
          character vectors as above, or numeric version objects.  For
          ‘is.numeric_version’ and ‘is.package_version’, arbitrary R
          objects.
```
So, our earlier usage was incorrect.

Reply:
makes sense.